frappe/public/js/frappe/views/image/image_view.js

```diff
@@ -65,12 +66,21 @@ frappe.views.ImageView = class ImageView extends frappe.views.ListView {
 		item._name = encodeURI(item.name);
 		const encoded_name = item._name;
 		const title = strip_html(item[this.meta.title_field || 'name']);
+		const info_fields = this.settings.imageview_fields || []
 		const _class = !item._image_url ? 'no-image' : '';
 		const _html = item._image_url ?
 			`<img data-name="${encoded_name}" src="${ item._image_url }" alt="${ title }">` :
 			`<span class="placeholder-text">
 				${ frappe.get_abbr(title) }
 			</span>`;
+		let info_html = `<div><ul class="list-unstyled image-view-info">`
```

Comment:
Template should come from `settings.render_image_view`

Reply:
settings.render_image_view as a function? @netchampfaris